commercetools-sdk-base/src/main/java/io/sphere/sdk/annotations/HasUpdateActions.java

```diff
@@ -0,0 +1,4 @@
+package io.sphere.sdk.annotations;
+
+public @interface HasUpdateActions {
```

Comment:
Please add some javadoc to this annotation, that describes that this annotation currently only enables the generation of update actions for primitive typed properties.